sensing/pointcloud_preprocessor/docs/image/concatenate_data.drawio.svg

```diff
@@ -1,4 +1,4 @@
LYk45Q2KX3+wd+CO88ZS/9b7X7ECnPlzWnSrokupdAU2prTqkvYTdofdCm1rT67mlPfxl99Sth/T/Ik2QohhBCZGZ4IL01/eJLtqrjXtFm3atvfHPxFkVKgisPPc9PU7w4+GEII4ev7f1Paoo7ka3W7lJC7crnpo++upPpVcfdZ9TFzfgvtu9yrZ/mSbIUQQoge0MNWCCGEyMzwofDK9Ie5OUQrd+TmsyrjYPlb+x+K1vvA2zODmrAuNfNhYn+8NUnc7XUac+wf6SYe8ziQBdiT+lm9qFF9jOUZ2lT9vLAqKt86bfpW75bVzXM8f26pTJKtEEIIkZk5A6lfuPv15RpdrTqxKnE2a7QZPLRVpCTNHlYGw60QQgj717ZmmezltsOF4gfDajHMk4B9e094p7R8us+OYED7SZxTZ+3bXB2sbptVX0HNqetpXqpqTPp0W2reXpKtEEII0Tl62AohhBCZGR4PN2e/zM22TZVuEzVy6jhDkpfSBtKDvUJ192mpjN+NfPOXZ/Gvf/5PK0YMY+dXlbqMHC6vOVWhqX2mHIeu1Mxtqnfb6qerudeYxyCQzxze54m69WrOqfXyhP9Xkq0QQgiRmTkDqdYk27Yk31Sjk4rtvv+H75um3/fV700Sp2OSqyTadzMfGXwSfs3SU9egJ6HYMxyqel7nDjCVwwiorbGb9sWO8TCSLuuzjnairJ/UNt74OfajLe1BrE3LEuNcu67mmVBPkq0QQgiRGT1shRBCiMwM9s/Ovmb/ycXJFqXiPdKIaTnq2DVEbJBKqWHLVNk+qi2+vP/qNP3q4MsdjSq6ZP/U1jR97mp/8xDp1LHDatp/2Zgxbban+a46pleeoplmz4Q1Z+zYOEzjy45NU823EZtn1eOdYlt2JmxN83Z3nw4hSLIVQgghsjM8d7HvKdSniX3W1h+BWfozLRk9SZo99AyubtH8L+6/EUII4Y3BF7ubjEgit6arqvdih8HGxAohyVYIIYTIzP8DL54DBOE4zOkAAAAASUVORK5CYII=" preserveAspectRatio="none"/><rect x="746" y="19" width="100" height="30" fill="none" stroke="none" pointer-events="all"/><g fill="rgb(0, 0, 0)" font-family="Helvetica" clip-path="url(#mx-clip-750-19-92-30-0)" font-size="12px"><text x="751.5" y="38.5">blockage mask</text></g><path d="M 726 48.08 L 839.63 48.08" fill="none" stroke="rgb(0, 0, 0)" stroke-miterlimit="10" pointer-events="stroke"/><path d="M 844.88 48.08 L 837.88 51.58 L 839.63 48.08 L 837.88 44.58 Z" fill="rgb(0, 0, 0)" stroke="rgb(0, 0, 0)" stroke-miterlimit="10" pointer-events="all"/><image x="728.38" y="-0.5" width="118.12" height="25.01" xlink:href="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAd0AAABlCAYAAAAI0znCAAAABHNCSVQICAgIfAhkiAAAABl0RVh0U29mdHdhcmUAZ25vbWUtc2NyZWVuc2hvdO8Dvz4AAAAzaVRYdENyZWF0aW9uIFRpbWUAAAAAADIwMjLlubQwM+aciDAx5pelIDE15pmCMDLliIYwMuenkisXU/oAAAIvSURBVHic7dzBSSRRFEDRp3QCpiBoNhNLM3uRykCMxWxmgqgQZlbCLFyI6P1j1zmrplaPgubyPsW/2vf9z8zM89PLsMbd/Fo9wsX7PferRwAO7Pzzx8zMXL1G9+bmeelAR7bNtnqEi+cdAyvt+3lmZq4XzwEAhyG6ABARXQCIiC4AREQXACKiCwAR0QWAiOgCQER0ASAiugAQEV0AiIguAEREFwAiogsAEdEFgIjoAkBEdAEgIroAEBFdAIiILgBERBcAIqILABHRBYCI6AJARHQBICK6ABA5rR4ACtts73oG8JVsugAQEV0AiIguAEREFwAiogsAEdEFgIjoAkBEdAEgIroAEBFdAIiILgBERBcAIqILABHRBYCI6AJARHQBICK6ABARXQCIiC4AREQXACKiCwAR0QWAiOgCQER0ASByWj0AM9ts73oGwPdm0wWAiOgCQMTx8n/AUTLAMdh0ASAiugAQEV0AiIguAEREFwAiogsAEdEFgIjoAkBEdAEg4kYqDuvfm8DcCgYUbLoAEBFdAIiILgBERBcAIj6kCvhIB4AZmy4AZGy6MG+fRjihAD6bTRcAIqILABHRBYCI6AJARHQBICK6ABARXQCIiC4AREQXACKiCwAR0QWAiOgCQER0ASAiugAQEV0AiIguAEREFwAiogsAEdEFgIjoAkBEdAEgIroAEBFdAIiILgBERBcAIqILAJHT64/t8XblHBftbh5Wj8AHbOM/AXyuv9B4HtK1IRUMAAAAAElFTkSuQmCC" preserveAspectRatio="none"/></g><switch><g requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"/><a transform="translate(0,-5)" xlink:href="https://www.diagrams.net/doc/faq/svg-export-text-problems" target="_blank"><text text-anchor="middle" font-size="10px" x="50%" y="100%">Text is not SVG - cannot display</text></a></switch></svg>
```

Comment:
[MUST] concatenate_data image is somehow changed to blockage image. Please revert it.